setup.cfg

```diff
@@ -0,0 +1,45 @@
+[metadata]
+name = pyamo
+version = 1.0.0
+author = Philipp Kewisch
+author_email = mozilla@kewis.ch
+url = https://github.com/kewisch/pyamo
+description = Access services on addons.mozilla.org from python
+long_description = file: ReadMe.md
+keywords = amo, mozilla, add-ons
+license = MPL-2.0
+classifiers =
+	Programming Language :: Python
+	Programming Language :: Python :: 3
+	License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)
+	Development Status :: 4 - Beta
+	Environment :: Console
+	Topic :: Utilities
+	Operating System :: OS Independent
+	Intended Audience :: Developers
+
+[options]
+zip_safe = True
+include_package_data = True
+packages = pyamo
+setup_requires = setuptools_scm;
+test_suite = tests.test
+dependency_links = git+https://github.com/benjaminp/six.git#egg=six, git+https://github.com/mozilla/PyFxA.git#egg=PyFxA, git+https://github.com/ahupp/python-magic.git, git+https://github.com/requests/requests.git#egg=requests, git+https://github.com/scrapy/cssselect.git#egg=cssselect, git+https://github.com/druths/arghandler.git#egg=arghandler, git+https://github.com/regebro/tzlocal.git#egg=tzlocal, git+https://github.com/dateutil/dateutil.git#egg=python-dateutil
```

Comment:
Can we make this multiline? Will this cause dependencies to be installed from github, or will they still be taken from pypi?

Reply:
In fact I have not managed make this work. I want the dependencies be installed from GitHub, but they still don't, don't know why. Even passing flags to pip doesn't help. We need to check if they have disabled this behavior. Even if the links are useless for auto install, I guess storing them here is useful anyway: I won't have to crawl the inet for the deps git repos the next time.

Reply:
I'd prefer to keep the dependencies on pypi.

Reply:
done